packages/mobile/src/screens/favorites-screen/PlaylistsTab.tsx

```diff
@@ -25,29 +32,21 @@ const messages = {
 export const PlaylistsTab = () => {
   const navigation = useNavigation<FavoritesTabScreenParamList>()
   const [filterValue, setFilterValue] = useState('')
-  const user = useSelector(getAccountWithPlaylists)
   const dispatch = useDispatch()
 
-  useEffectOnce(() => {
+  const handleFetchSavedPlaylists = useCallback(() => {
     dispatch(fetchSavedPlaylists())
-  })
+  }, [dispatch])
 
-  const matchesFilter = (playlist: ExtendedCollection) => {
-    const matchValue = filterValue.toLowerCase()
-    return (
-      playlist.playlist_name.toLowerCase().indexOf(matchValue) > -1 ||
-      playlist.ownerName.toLowerCase().indexOf(matchValue) > -1
-    )
-  }
+  useEffectOnce(handleFetchSavedPlaylists)
```

Comment:
I cant use `usefocusEffect` since this tab would lazy load, and we don't have a good loading indicator for this (fetching collections doesn't have a status associated with it) this would be a good improvement though eventually, since might as well reduce fetches as much as we can